systemtest/playlist.xml

```diff
@@ -1451,7 +1467,7 @@
 		<impls>
 			<impl>openj9</impl>
 		</impls>
-		<platformRequirements>os.linux,^arch.ppc</platformRequirements>
+		<platformRequirements>os.linux,^arch.ppc,^os.win</platformRequirements>
```

Comment:
you don't need to exclude this test, as it's linux only

Reply:
Rght! Thanks for spotting. There were few more of these. Updated them all.